src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java

```diff
@@ -189,4 +190,20 @@ public BufferedImage update(PlayerSetupInfo info)
       return myImage;
     }
   }
+
+  /**
+   * Draws little white boxes around all the currently-highlighted player's properties.
+   * @param distance - defines how far the boxes should be from the property edge (serves to highlight the new properties when switching between players)
+   */
+  private static void drawPlayerPropertyHighlights(Graphics g, int baseX, int baseY, MapInfo mapInfo, int scale, int faction, int distance)
```

Comment:
This is nice.

Reply:
Thanks.
It was inspired by going "wait, which team was that, again?" on a 14 player map.